notebooks/1.1-vse-eda-rooms.ipynb

```diff
@@ -1,943 +0,0 @@
-{
```

Comment:
This pull request is about data cleanning. So, this file shouldn't be here.

Reply:
Yeah, I realized it was there afterwards and deleted it lol